torchtext/experimental/datasets/text_classification.py

```diff
@@ -32,7 +32,7 @@ def build_vocab(data, transforms):
     return build_vocab_from_iterator(tok_list)
 
 
-def squential_transforms(*transforms):
+def sequential_transforms(*transforms):
```

Comment:
You could send out a separate PR to fix those typos together with `start` and `num_lines` in `TextClassificationDataset`

Reply:
Sure. I'll revert these changes and submit another PR.